content/dev/dev-guide.md

```diff
@@ -0,0 +1,325 @@
+    - For the glob, we've used a clause of `%glob-ames`, which means the glob will be served from a ship over Ames, as opposed to being served over HTTP with a `%glob-http` clause or having an Eyre binding with a `%site clause`. 
+    - Next we've specified `[~zod 0v0]`. Since `~zod` is the fakeship we'll install it on, the `%docket` agent will wait for a separate upload of the glob, so we can just specify `0v0` here as it'll get overwritten later.
+    - The `%version` clause specifies the version as a triple of major version, minor version and patch version. The rest is just some additional informative metadata which will be displayed in App Info.
+- With that saved we now have everything we need in the `hello` directory.
+
+##### Committing and Installing The Changes
+
+ - Let's start up fake zod.
+   - ```bash
+     cd ~/urbit-dev/
+     ./urbit zod
+     ```
+- Commit and install our changes.
+  - ```bash
+    |commit %hello
+    |install our %hello
+    ```
+  - You'll see `docket: awaiting manual glob for %hello desk` this is because our `desk.docket-0` files includes a `%glob-ames` clause which specifies our ship as the source, so it's waiting for us to upload the glob.
+- Open a browser and navigate to `http://localhost:80`. To get the login code, enter `+code` into the dojo prompt for your ship. In this case, the default is `lidlut-tabwed-pillex-ridrup`. You'll see our tile, but it says installing with a spinner due to the missing glob.
+
+##### Create Files for Glob
+
+We'll now create the files for the glob. We'll use a very simple static HTML page that just displayes "Hello World!" and an image. Typically we'd have a more complex JS web app that talked to apps on our ship through Eyre's channel system, but for the sake of simplicity we'll forgo that. Let's hop back in the Unix terminal.
+ - ```bash
+   cd ~
+   mkdir hello-glob
+   cd hello-glob
+   mkdir img
+   curl https://media.urbit.org/docs/userspace/dist/pot.svg --output img/pot.svg
+   ```
+- We've grabbed an image to use in our "Hello world!" page. The next thing we need to add is an `index.html` file in the root of the folder. The `index.html` file is mandatory; it's what will be loaded when the app's tile is clicked. Let's open our preferred editor and create it.
+  - ```bash
+    nano index.html
+    ```
+- Paste the following into the editor and save it.
+  - ```html
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Hello World</title>
+        <style>body { text-align: center; }</style>
+      </head>
+      <body>
+        <h1>Hello World!</h1>
+        <img src="img/pot.svg" alt="pot" width="219" height="196" />
+      </body>
+    </html>
+    ```
+- **Note**: Why are we using globs to hold separate files instead of copying everything into Clay?
+  - We do this as a current loom limits work around.
```

Comment:
Recommend not mentioning `loom`, just `VM memory limits`.

Reply:
This and the next three lines should be a single item or paragraph rather than separate entries.

Reply:
Thanks - I should have time to refactor all of this today and push an update